testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -291,6 +296,26 @@ def test_write_single_frame_AtomGroup(self, universe2, outfile):
                                                "agree with original coordinates from frame %d" %
                                                u.trajectory.frame)
 
+    def test_write_nodimes(self, universe4, outfile):
```

Comment:
More of a thought, please do ignore; would `nodims` be more descriptive than `nodimes` here?

Reply:
Probably more worth it, maybe consider adding some slight docstring to point out the issue this is testing for, as per the above test?

Reply:
This is actually a typo. Adding a reference to the issue in the docstring is a great idea!